src/components/Modals/MapModal.vue

```diff
@@ -19,7 +19,7 @@ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
 
             <button @click="$refs.importInput.click()" id="import">Open Floorplan</button>
             <button @click="mapEnabled = false; mapVisible = false; $emit('close')">New Floorplan</button>
-            <button @click="mapEnabled = true; tool='Map'; $emit('close')">New Floorplan w/ Map</button>
+            <button @click="mapEnabled = true; tool='Map'; $emit('close')" :disabled="!online">New Floorplan w/ Map</button>
```

Comment:
I totally agree. But we're able to write css to target disabled buttons:

```
button[disabled] {
   cursor: not-allowed;
/* hatch marks? open to whatever, but thought this might look nice */
  background-image: 
    linear-gradient(-45deg, transparent 45%, #919195 50%, transparent 55%, transparent 100%);
  background-size: 8px 8px;
  background-position: 4px 4px;

}
```

I think it would be good if the disabled buttons were visually distinct.